### ISSUE TITLE
Error in Accessibility Statement detection

### ISSUE THREAD

**Issue by mgifford** [mgifford/daily-dap#181]
The report says that VA doesn’t have an accessibility statement, but it does: https://www.va.gov/accessibility-at-va/

I can see that it exists here in the report:

va.gov | ✓ Found | https://va.gov/accessibility 

but this is a 404, as it redirects to:

https://www.va.gov/accessibility

but it is redirected here:

www.va.gov | ✗ Not found

I think that we're not measuring the redirects properly.

The www & non-www domains are almost always the same. there will be one, they might both work, but by convention they should have the same content, so should get the same results. 

I think the redirect is the problem here. 

https://va.gov/

redirects to:
https://www.va.gov/

Somehow this whole search seems to be very screwed up.

I am also trying to do something similar, for more languages here:
https://github.com/mgifford/eu-plus-government-scans

The approach is defined here:
https://mgifford.github.io/eu-plus-government-scans/accessibility-statements.html

Maybe something there could be replicated.
